### PR TITLE
[78161] Show fastseg models in OMZ table

### PR DESCRIPTION
**Pull request by vgoluben**
### All Submissions:
* [x] Create a meaningful name of the PR with reference to JIRA. Example: *[43714] [NIGHTLY] Take correct tag for snyk scans*
* [x] JIRA tickets:
  * 78161 - Add Cityscapes-trained models to OMZ download table
* [x] Set assignee to the PR.
* [x] Add corresponding labels to the PR. 
* [x] Set appropriate milestone to the PR.
* [x] Added tests for new functionality (unit / e2e)
* [x] Followed code style guidelines:
  * Code covered with typings
  * No debug printing in production code
  * No commented blocks of code in production code

### Submissions with migration changes:
* [x] Check that workbench still works after the update from the previous version

### CODE REVIEW

File: migrations/versions/9ede8e012999_enable_fastseg_models.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+def upgrade():
+
+    op.execute(
+        'UPDATE omz_topologies SET advanced_configuration = \'{"preprocessing": [{"type": "auto_resize"}], "postprocessing": [{"type": "encode_segmentation_mask", "apply_to": "annotation"}, {"type": "resize_segmentation_mask", "apply_to": "prediction"}], "metric": [{"type": "mean_iou", "use_argmax": false, "presenter": "print_scalar", "reference": 0.7267}], "annotationConversion": {}, "adapterConfiguration": {"adapter": {"type": "segmentation", "make_argmax": true}}}\' WHERE name = \'fastseg-large\';')
+    op.execute(
+        'UPDATE omz_topologies SET advanced_configuration = \'{"preprocessing": [{"type": "auto_resize"}], "postprocessing": [{"type": "encode_segmentation_mask", "apply_to": "annotation"}, {"type": "resize_segmentation_mask", "apply_to": "prediction"}], "metric": [{"type": "mean_iou", "use_argmax": false, "presenter": "print_scalar", "reference": 0.6715}], "annotationConversion": {}, "adapterConfiguration": {"adapter": {"type": "segmentation", "make_argmax": true}}}\' WHERE name = \'fastseg-small\';')
+
+
+def downgrade():
+    raise NotImplementedError(f'Downgrade is not implemented for the {revision} migration')
```
Comment (artyomtugaryov) [inline migrations/versions/9ede8e012999_enable_fastseg_models.py:40]: new line

File: migrations/versions/9ede8e012999_enable_fastseg_models.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ You may obtain a copy of the License at
+      https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '9ede8e012999'
+down_revision = 'f7fcb58b99cd'
```
Comment (artyomtugaryov) [inline migrations/versions/9ede8e012999_enable_fastseg_models.py:26]: Have you checked the order?

Comment (vgoluben) [reply migrations/versions/9ede8e012999_enable_fastseg_models.py:26]: Yes, the order has already been adjusted to that of master

**Review by artyomtugaryov — APPROVED** [review]
Need to check migration order before merge

**Pull request merged**